jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/VolumePermissionEnricher.java

```diff
@@ -178,6 +187,33 @@ private String getMountPath(List<Container> containers, String name){
                 throw new IllegalArgumentException("No matching volume mount found for volume "+ name);
             }
 
+            private ResourceRequirements createNewResourceRequirements() {
+                if (hasNoResourcesConfigured())
+                    return null;
+                ResourceRequirementsBuilder resourceRequirementsBuilder = new ResourceRequirementsBuilder();
+                resourceRequirementsBuilder.withLimits(createResourcesMap(Config.CPU_LIMIT, Config.MEMORY_LIMIT));
+                resourceRequirementsBuilder.withRequests(createResourcesMap(Config.CPU_REQUEST, Config.MEMORY_REQUEST));
```

Comment:
Is there a typo in the config properties used?

Reply:
I'm not sure about this.

Reply:
It was confusing for me in the context of the other comments. Seems fine now.